### ISSUE TITLE
Pub/Sub requirement seems to be overkill

### ISSUE THREAD

**Issue by netdata-be** [MyTeslaMate/websocket#2]
Currently this project relies to the following:

* Tesla Telemetry reference implementation => Pub/Sub
* PubSub HTTP post => this websocket project
* Teslamate ( or any other legacy streaming API ) => this web socket project

I think the hurdle ( and the costs) of having Pub/Sub is overkill for most projects, It would make more sense to directly do an HTTP post towards this project.
There is a PR open to implement a HTTP dispatcher.

https://github.com/teslamotors/fleet-telemetry/pull/91

Would create a cleaner integration, the issue with the HTTP dispatcher is that it is does not support ACK, but currently I think having the Pub/Sub http post does not ack the messages as well, so no difference here.

**Comment by jlestel** [comment]
I agree, it's overkill, but it was the quickest way for me while waiting for the http dispatcher PR you mentioned to be merged.

**Comment by aksvenk** [comment]
Not sure if there is any movement on that PR. Are there any alternatives which are fully self hosted?

**Comment by andrew-kennedy** [comment]
Now that there is an MQTT dispatcher for the tesla-fleet telemetry server (see release v0.6.0) can we get MQTT support in this telemetry server? or anything really to avoid having to host our own "cloud" pubsub? I have an MQTT broker (same one teslamate is connected to actually) already receiving all tesla fleet telemetry events. Was hoping to use teslamate to "transform" them into home assistant devices as the teslamate integration is already more mature.

**Comment by jlestel** [comment]
Yes, i'll work on it.

Feel free to init some PR if you have nodejs skills.

**Comment by dwery** [comment]
+1 for MQTT!
